core/src/test/java/kafka/log/remote/RemoteLogReaderTest.java

```diff
@@ -57,7 +56,7 @@ public void setUp() {
         TestUtils.clearYammerMetrics();
         Properties props = kafka.utils.TestUtils.createDummyBrokerConfig();
         props.setProperty(RemoteLogManagerConfig.REMOTE_LOG_STORAGE_SYSTEM_ENABLE_PROP, "true");
-        brokerTopicStats = new BrokerTopicStats(Optional.of(KafkaConfig.fromProps(props)));
+        brokerTopicStats = new BrokerTopicStats(KafkaConfig.fromProps(props).isRemoteLogStorageSystemEnabled());
```

Comment:
Seems the `props` here just created to pass a single config and is not required now. We might just require a single line below with removed `props`.

```
 brokerTopicStats = new BrokerTopicStats(true);
```

Reply:
Yes that's a good idea. Done!